src/components/NetlifyForm/index.jsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * NetlifyForm component consists of server related functionalities that is used posting data and also avoiding captchas.
+ * The base code that is used in both netlifyForm and index are implemented from https://github.com/jon-fm/react-ssg-netlify-forms-demo
+ */
+import React, { useState, useEffect } from "react";
+
+const NetlifyForm = ({
+  formName,
+  beforeSubmit,
+  afterSubmit,
+  contactFormValues,
+  children,
```

Comment:
You should clarify use and types of these parameters inside comments

Reply:
Thanks Tarkan, I wrote inline notes for them.